### PR TITLE
[TA] Added extractive summarization links in READMEs

### PR DESCRIPTION
**Pull request by mssfang**
Revisited READMEs and added extractive summarization links.

### CODE REVIEW

**Review by kristapratico — APPROVED** [review]

File: sdk/textanalytics/azure-ai-textanalytics/README.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ and includes the main functions below:
 - Linked Entity Recognition
 - Healthcare Entity Recognition
 - Multiple Actions Analysis Per Document
+- Extractive Text Summarization
```
Comment (deyaaeldeen) [inline sdk/textanalytics/azure-ai-textanalytics/README.md:13]: it is not a function though

**Review by samvaity — APPROVED** [review]

File: sdk/textanalytics/azure-ai-textanalytics/README.md (hunk 1 above)
Comment (mssfang) [reply sdk/textanalytics/azure-ai-textanalytics/README.md:13]: We should mention that we have this feature. 

Trying to keep consistency with .NET and Python 
.net: https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/textanalytics/Azure.AI.TextAnalytics
python:  https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/textanalytics/azure-ai-textanalytics/README.md

Comment (deyaaeldeen) [reply sdk/textanalytics/azure-ai-textanalytics/README.md:13]: Yeah let's try to be consistent but also correct. @maririos what do you think about updating the text above to `and includes the main features below`?

Comment (maririos) [reply sdk/textanalytics/azure-ai-textanalytics/README.md:13]: sounds good with me!

Comment (mssfang) [reply sdk/textanalytics/azure-ai-textanalytics/README.md:13]: @kristapratico FYI Since we are changing the keyword `functions` to `features`.

Comment (deyaaeldeen) [reply sdk/textanalytics/azure-ai-textanalytics/README.md:13]: I made the change for JS in https://github.com/Azure/azure-sdk-for-js/pull/16789

Comment (kristapratico) [reply sdk/textanalytics/azure-ai-textanalytics/README.md:13]: I got in this PR: https://github.com/Azure/azure-sdk-for-python/pull/20115/files

**Pull request merged**